templates/js/copilot-plugin-from-scratch/README.md

```diff
@@ -24,22 +24,23 @@ The plugin allows Teams to interact directly with third-party data, apps, and se
 
 ## What's included in the template
 
-| Folder       | Contents                                     |
-| ------------ | -------------------------------------------- |
-| `.vscode`    | VSCode files for debugging                   |
-| `appPackage` | Templates for the Teams application manifest |
-| `env`        | Environment files                            |
-| `infra`      | Templates for provisioning Azure resources   |
-| `repair`     | The source code for the repair API           |
+| Folder       | Contents                                                                                                |
+| ------------ | ------------------------------------------------------------------------------------------------------- |
+| `.vscode`    | VSCode files for debugging                                                                              |
+| `appPackage` | Templates for the Teams application manifest, the API specification and Adaptive Card for API responses |
```

Comment:
wondering if we want to say "response templates“ rather than "Adaptive cards" @MuyangAmigo thoughts?

Reply:
Agreed.